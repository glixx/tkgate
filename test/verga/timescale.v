`timescale 1ns/10ps

module top;
  foo f1;
  bar b1;

  initial
    #5.5 $display("%t (%d): top",$time,$time);

endmodule

`timescale 100ps/10ps

module foo;

  initial
    #5 $display("%t (%d): foo",$time, $time);

endmodule

`timescale 10ps/10ps

module bar;

  initial
    #5 $display("%t (%d): bar",$time, $time);

endmodule
