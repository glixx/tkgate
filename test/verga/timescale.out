5 (5): bar
5.0 (50): foo
5.50 (550): top
